plugins/linear-genome-view/src/BasicTrack/baseTrackModel.tsx

```diff
@@ -74,7 +84,60 @@ const generateBaseTrackConfig = (base: any) =>
         defaultValue: [],
       },
     },
-    { explicitIdentifier: 'trackId' },
+    {
+      explicitIdentifier: 'trackId',
+      views: (self: unknown) => ({
+        get fileMenuItems() {
+          let session: any
+          const config = self as IAnyStateTreeNode
+          try {
+            session = getSession(config)
+          } catch (e) {
+            // TODO likely want a better way to do this
+            session = getRoot(config).session
```

Comment:
Maybe we just make getSession smarter to handle this situation.

Reply:
I made the logic for this move to session

Reply:
Called session.getTrackActionMenuItems(trackConf)